tests/src/JIT/HardwareIntrinsics/X86/Sse/StaticCast.cs

```diff
@@ -39,12 +47,43 @@ static unsafe int Main(string[] args)
                         Console.WriteLine();
                         testResult = Fail;
                     }
-                }
-            }
 
+                    // the successful path is the one that catches the exception, so that does nothing, 
```

Comment:
nit: it might be clearer if you put this comment in each catch block (or something similar to indicate that we expect the exception to be thrown.